### PR TITLE
Cleanup repository

### PR DESCRIPTION
**Pull request by khanguslee**
Also DAS scripts now print out a short decription of what program it is. This allows for a team member to view what script is currently loaded onto the Teensy.

Removed some trailing whitespaces too.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,5 @@
 .vscode
 
 node_modules/
-.eslintrc.*
\ No newline at end of file
+.eslintrc.*
```
Comment (hallgchris) [inline .gitignore]: Why don't we want .eslintrc? I'd have thought it should be here for CI and so we're all following the same standards

**Review by hallgchris — APPROVED** [review]
LGTM

File: .gitignore (hunk 1 above)
Comment (khanguslee) [reply .gitignore]: I swear this was added in automatically LMAO. I'll remove this.

**Review by harsilspatel — APPROVED** [review]
yeah good work @khanguslee, merging it ^_^

**Pull request merged**